journal/modernized/02-chapter-1.adoc

```diff
@@ -82,14 +82,16 @@ A young man, a merchant, then present, who frequented my company,
 and would often speak of the people called Quakers,
 and say their principles were the truth,
 hearing the priest boast of his abusive behaviour to the said two women,
-answered and said.
-It was a poor victory he had gotten over two poor women;
+answered and said,
+"`It was a poor victory he had gotten over two poor women;`"
+//There aren't quotations in the PDF, but this seems like a direct quote to me. Also on line 94
+
 at which the priest was very angry, and began to storm.
 My spirit rose against him; I started up from my seat,
-and asked the priest and them with him, if they came to quarrel?
+and asked the priest and them with him, if they came to quarrel,
 saying, if they did, they should have enough.
-The priest answered.
-No, not with you sir.
+The priest answered,
+"`No, not with you sir.`"
```

Comment:
I agree here....  this looks like a direct quote. I know its tricky with these old documents, they weave in and out of direct quotes and narration a lot. I'm running into the same problems in the translation process into Spanish.